go.mod

```diff
@@ -139,6 +139,7 @@ require (
 	k8s.io/helm v2.9.0+incompatible
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v0.0.0
+	k8s.io/kubernetes v1.17.3
```

Comment:
We've been working towards dropping k8s.io/kubernetes as a dependency (example: https://github.com/kubernetes/kops/pull/8055 and https://github.com/kubernetes/kops/pull/8056) since it can be a nightmare to update with each minor version and all of the transitive dependencies.

Seeing as you're only using the `ParseTaints` function and there doesn't appear to be any effort to move it to one of the other k8s.io repos, maybe we consider copying the function directly into the kops repo? It looks like ParseTaints and two unexported functions might be all that is needed. 

@justinsb what do you think? You can also provide more context on shifting away from depending on k/k.

Reply:
I've removed the `k8s.io/kubernetes` dependency and [extracted the relevant parts only](https://github.com/kubernetes/kops/blob/2a17a50d007f3f0ef9f8fb1598838ab58fbbd99d/pkg/model/spotinstmodel/instance_group.go#L847-L883).